packages/ui/v2/core/Shell.tsx

```diff
@@ -38,9 +38,9 @@ import {
 import ErrorBoundary from "../../ErrorBoundary";
 import { KBarContent, KBarRoot, KBarTrigger } from "../../Kbar";
 import Logo from "../../Logo";
+import { SkeletonText } from "../../components/skeleton";
```

Comment:
long term this will be @calcom/components right?

Reply:
In general best practice within the UI folder itself is to use relative imports for everything. That will drastically reduce the amount of recursive imports. So in my opinion this is still up for debate.